src/Pagination/PhalconFrameworkPaginatorAdapter.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+/*
+ * This file is part of the League\Fractal package.
+ *
+ * (c) Phil Sturgeon <me@philsturgeon.uk>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace League\Fractal\Pagination;
+
+/**
+ * A paginator adapter for PhalconPHP/pagination.
+ *
+ * @author Thien Tran <fcduythien@gmail.com>
+ * 
+ */
+class PhalconFrameworkPaginatorAdapter implements PaginatorInterface
+{
+    /**
+     * A slice of the result set to show in the pagination
+     *
+     * @var \Phalcon\Paginator\AdapterInterface
+     */
+    protected $paginator;
```

Comment:
I would actually make this a private property, unless there's significant reason for it to be protected.